processor/transformer/transformer.go

```diff
@@ -444,12 +444,21 @@ func (trans *handle) doPost(ctx context.Context, rawJSON []byte, url, stage stri
 
 	err := backoff.RetryNotify(
 		func() error {
-			var reqErr error
 			requestStartTime := time.Now()
 
-			trace.WithRegion(ctx, "request/post", func() {
```

Comment:
Took some liberty and got rid of this trace. 
a. This is causing lint issue
b. We are not using anyways